molecule/default/verify.yml

```diff
@@ -73,3 +73,52 @@
       - not example_app.changed
       msg: |
         Re-cloning example app resulted in changed status
+
+  # Testing dokku_network
+  - name: Create a network # noqa 301
+    dokku_network:
+      name: example-network
+    register: example_network
+
+  - name: Get list of networks # noqa 301
+    command: dokku network:list
+    register: dokku_networks
+
+  - name: Check that example-network is in list of networks
+    assert:
+      that:
+      - "'example-network' in dokku_networks.stdout"
+      msg: |-
+        'example-network' not found in output of 'dokku network:list':
+        {{ dokku_networks.stdout }}
+
+  - name: Create a network that already exists # noqa 301
+    dokku_network:
+      name: example-network
+    register: example_network
+
+  - name: Check that re-creating network example-network did not change anything
+    assert:
+      that:
+      - not example_network.changed
+      msg: |
+        Re-creating network example-network resulted in changed status
+
+  # Testing dokku_network_property
+  - name: Setting a network property for an app
+    dokku_network_property:
+      app: hello-world
```

Comment:
I think you want to go with `example-app` here, which is created in the lines above

Reply:
Yeah, wasn't the only problem... I had a hard time getting molecule test getting to run on my local system properly. Neverless, tests are working now.

Reply:
> I had a hard time getting molecule test getting to run on my local system properly. Neverless, tests are working now.

Ah - I mostly just rely on the CI test setup myself.
What are the issues you ran into? Would be happy to add them to the [contributing docs](https://github.com/dokku/ansible-dokku/blob/master/CONTRIBUTING.md) in a separate PR.

Reply:
The thing is that molecule does not tell my on my local setup what's actually wrong:

I```
NFO     Running default > verify
INFO     Running Ansible Verifier

PLAY [Verify] ******************************************************************

TASK [Gathering Facts] *********************************************************
ok: [instance]

TASK [Run role without running any tasks] **************************************
ERROR! Could not find specified file in role: tasks/init.yml

PLAY RECAP *********************************************************************
instance                   : ok=1    changed=0    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0

CRITICAL Ansible return code was 2, command was: ansible-playbook --inventory /Users/lemoony/.cache/molecule/ansible-dokku/default/inventory --skip-tags molecule-notest,notest /Users/lemoony/projects/opensource/ansible-dokku/molecule/default/verify.yml
WARNING  An error occurred during the test sequence action: 'verify'. Cleaning up.
INFO     Running default > cleanup
WARNING  Skipping, cleanup playbook not configured.
INFO     Running default > destroy
```

So I have to push the code each time to see what's wrong...